test/unit/test_cluster_fuzz.py

```diff
@@ -421,7 +421,8 @@ def test_file_contents(self):
             for i in range(1, N + 1):
                 fuzz_file = os.path.join(temp_dir.name, f'fuzz-binaryen-{i}.js')
 
-                cmd = [shared.V8, '--wasm-staging', fuzz_file]
+                # Add --fuzzing to allow legacy and standard EH to coexist
+                cmd = [shared.V8, '--wasm-staging', '--fuzzing', fuzz_file]
```

Comment:
This is probably an unrelated side effect of having updated my local V8 version.